src/lib/swapper/swappers/LifiSwapper/LifiSwapper.ts

```diff
@@ -10,6 +10,8 @@ import {
   filterSameChainEvmBuyAssetsBySellAssetId,
 } from '../utils/filterBuyAssetsBySellAssetId/filterBuyAssetsBySellAssetId'
 
+export const TRADE_POLL_INTERVAL_MILLISECONDS = 30_000
```

Comment:
Could be worth qualifying this with lifi so its clear it doesnt apply to other swappers (even though its in a lifi swapper directory)

```suggestion
export const LIFI_TRADE_POLL_INTERVAL_MILLISECONDS = 30_000
```

Reply:
Jinxed 😄 https://github.com/shapeshift/web/pull/6992/files#r1618387853
Could be an option too instead of import as

Reply:
Good call, updated const naming: https://github.com/shapeshift/web/pull/6992/commits/0c8707abcc6a2958889dc8346a8c92c87fee2ce6